server/src/utils/parseResponse.ts

```diff
@@ -23,18 +23,16 @@ export const parseResponse = (jsonResponse: SocialMediaComments): ResponseData =
   };
 
   for (const key in jsonResponse) {
-    if (jsonResponse.hasOwnProperty(key)) {
```

Comment:
@viiiiiicccky can check the use of `hasOwnProperty`?
i removed this due to eslint, but i wasn't really sure about how the check works -- it seems duplicated since we're already looping through all the keys of the object (and those keys are guaranteed to be properties of the object)?
let me know if I'm missing something, thanks!

Reply:
yes it should be safe to remove this! thanks for cathing!